docs/src/pages/getting-started/faq/faq.md

```diff
@@ -372,7 +373,7 @@ function Portal({ children, container }) {
 With this simple heuristic `Portal` might re-render after it mounts because refs are up-to-date before any effects run.
 However, just because a ref is up-to-date doesn't mean it points to a defined instance.
 If the ref is attached to a ref forwarding component it is not clear when the DOM node will be available.
-In the above example the `Portal` would run run an effect once but might not re-render because `ref.current` is still `null`. This is
+In the above example, the `Portal` would run an effect once but might not re-render because `ref.current` is still `null`. This is
```

Comment:
```suggestion
In the example above, the `Portal` would run an effect once, but might not re-render because `ref.current` is still `null`. This is
```